client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx

```diff
@@ -49,6 +49,28 @@ const ProcessingStep: React.FC< ProcessingStepProps > = function ( props ) {
 	const [ hasEmptyActionRun, setHasEmptyActionRun ] = useState( false );
 	const [ destinationState, setDestinationState ] = useState( {} );
 
+	/**
+	 * There is a long-term bug here that the `submit` function will be called multiple times if we
+	 * call `resetOnboardStoreWithSkipFlags` after the submit function (e.g.: exitFlow) to reset states
```

Comment:
Note to myself: like [here](https://github.com/Automattic/wp-calypso/blob/14350177998d9b455eb3291083dc7076b0bc2b18/client/landing/stepper/declarative-flow/site-setup-flow.ts#L250)